src/bindings/python/flux/uri/resolvers/lsf.py

```diff
@@ -0,0 +1,89 @@
+###############################################################
+# Copyright 2022 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+
+"""
+This module supports the Flux URI resolver for IBM Spectrum LSF.
+
+It can be used like:
+  `flux uri --remote lsf:<JOBID>`.
+
+"""
+
+import os
+import getpass
+import subprocess
+from pathlib import PurePath
+
+import yaml
+
+import flux.hostlist as hostlist
+from flux.uri import URIResolverPlugin
+
+def lsf_find_compute_node(jobid):
+    """Figure out where the job is being run using YAML output from IBM Cluster Systems Manager."""
+
+    if os.getenv("CSM_ALLOCATION_QUERY"):
+        csm_path = str(os.getenv("CSM_ALLOCATION_QUERY"))
+    else:
+        csm_path = "/opt/ibm/csm/bin/csm_allocation_query"
```

Comment:
The above four lines could be simplified to: `csm_path = os.getenv("CSM_ALLOCATION_QUERY", "/opt/ibm/csm/bin/csm_allocation_query")`.